doc/radosgw/lua-scripting.rst

```diff
@@ -42,14 +50,14 @@ To print the content of the script to standard output:
 
 ::
    
-   # radosgw-admin script get --context={preRequest|postRequest|background} [--tenant={tenant-name}]
+   # radosgw-admin script get --context={preRequest|postRequest|background|getData} [--tenant={tenant-name}]
```

Comment:
Here and below we have a mix of camelCase and lowercase.  I suggest a consistent practice throughout the document, whichever is context-appropriate.

Reply:
moved everything to lowercase (as with the rest of the radosgw-admin commands)